### DISCUSSION TITLE
SSL Pinning With Frida On iOS AppStore

### DISCUSSION THREAD

**Discussion by senolatac** [mitmproxy/mitmproxy#5933] [Intercept Everything!]
#### Problem Description
App store network can be listened with (Jailbreak + Frida + Objection) except some endpoints like (search-tab ads: https://searchads.apple.com/help/ad-placements/0071-search-tab) but I don't understand why. Is there any suggestion or guess here?

mitmproxy detect each network item but except this one: https://searchads.apple.com/help/ad-placements/0071-search-tab.

#### Steps to reproduce the behavior:
1. Jailbreak with Checkra1n or Palera1n...
2. Then trace traffic with https://zeroinfosec.com/2020/05/07/the-only-way-to-bypass-ssl-pinning-on-ios-13/
3. Detect network with mitmproxy

#### System Information
```
Mitmproxy: 9.0.1
Python:    3.11.1
OpenSSL:   OpenSSL 1.1.1s  1 Nov 2022
Platform:  macOS-12.5-x86_64-i386-64bit
```

**Comment by 3052** [comment]
I found recently that MITM Proxy is failing with Frida, but if you use the same exact script with HTTP Toolkit it works. I prefer MITM Proxy, but if it doesn't work I have no choice but to use something else
